CMakeLists.txt

```diff
@@ -101,7 +101,7 @@ else(MSVC)
     set(CMAKE_C_FLAGS "-O2 -Wall -fPIC ${CMAKE_C_FLAGS}")
     set(CMAKE_CXX_FLAGS "-O2 -Wall -fPIC ${CMAKE_CXX_FLAGS}")
     if (HIDE_PRIVATE_SYMBOLS)
```

Comment:
@tqchen noted that this config flag doesn't follow `USE_XXX` pattern, is that intended?